csvcubed/csvcubed/utils/cache.py

```diff
@@ -1,6 +1,7 @@
 from requests_cache import CachedSession
 
-def session(url):
+def start_session(url):
```

Comment:
I think it'd be better if you just exposed the `CachedSession` instead of making this function available. Code elsewhere can then call `cached_session.get("http://my-url")` quite naturally.